scene/gui/line_edit.cpp

```diff
@@ -357,8 +357,8 @@ void LineEdit::_gui_input(Ref<InputEvent> p_event) {
 			}
 		}
 
-		// Default is ENTER, KP_ENTER. Cannot use ui_accept as default includes SPACE
-		if (k->is_action("ui_text_newline", true)) {
+		// Default is ENTER. Cannot use ui_accept as default includes SPACE
+		if (k->is_action("ui_text_search_next", true) || k->is_action("ui_text_search_previous", true)) {
```

Comment:
This is tricky, as search is misleading.

I'm thinking about `ui_text_submit` and renaming the signal to `text_submitted` so they align.

Reply:
What are your thoughts about naming it to `ui_text_entered`? I'm a bit hesitant to rename the signal to `text_submitted`, since a lot of other functions are connecting to `text_entered`

Reply:
Yeah the problem with `ui_text_entered` is that it somewhat implies that text has been entered, rather then the `enter` key has been pressed. Given that we're making it a configurable shortcut, it may not event be the `enter` key, so would be even more confusing.

Reply:
Agreed, it's definitely not ideal. So you think I should rename the signal from `text_entered` to `text_submitted` and update all the files that use `LineEdit`? Or would it be better to just add a new signal called `text_submitted`?

Reply:
Go for the rename, this is for 4.0 only so it's okay to break compat. 

Though probably worth doing the rename as a separate commit.

Reply:
Okay. I will keep the signal name `text_entered` for this PR. I will mark this comment resolved once I submit a follow on PR that renames `text_entered` to `text_submitted`

Reply:
Forget the extra PR. I forgot we can squash the commit after the review if everything gets approved